src/pages/Home.js

```diff
@@ -64,6 +64,16 @@ const Home = () => {
       showOldPrice: false,
       ratersNumber: 43,
     },
+    {
+      image: "../assets/newArrivals/pink-bag-small.png",
```

Comment:
assets path should be /assets/newArrivals/pink-bag-small.png instead of ../assets/newArrivals/pink-bag-small.png because assets folder is in the public folder